### DISCUSSION TITLE
In App Live Preview Duplicating

### DISCUSSION THREAD

**Discussion by treeguy1234** [phcode-dev/phoenix#1478] [General]
When I first pull up the in app live preview, it is fine(see picture 1), but the when I edit the html the live preview duplicates(see picture 2)
What sould I do about it? Can you fix it?

<img width="367" alt="image" src="https://github.com/phcode-dev/phoenix/assets/125884868/980420cb-402b-4461-8b3c-48829b53cd0b">
<img width="362" alt="image" src="https://github.com/phcode-dev/phoenix/assets/125884868/e7ad887b-dd2d-4191-b268-6ceefd287d5e">

**Comment by abose** [comment]
Thanks for reporting the issue. This happens in rare cases with specific html files. Can you send over a sample html that has the issue and steps to investigate.

**Reply by treeguy1234** [reply]
It is fixed now
